stage7/02-cosmetic/00-run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+cp -f files/wallpaper.png "${ROOTFS_DIR}/usr/share/rpd-wallpaper"
+cp -f files/launch.png "${ROOTFS_DIR}/usr/share/raspberrypi-artwork"
+
+on_chroot << EOF
+sed -i 's+/usr/share/rpd-wallpaper/temple.jpg+/usr/share/rpd-wallpaper/wallpaper.png+g' /etc/xdg/pcmanfm/LXDE-pi/desktop-items-0.conf
```

Comment:
what happens when this changes later? and it's not `temple.jpg`
is there some other way to configure this without needed to explicitly know the file?